OrbitBase/include/OrbitBase/Tracing.h

```diff
@@ -16,7 +16,7 @@
 
 #define ORBIT_SCOPE_FUNCTION ORBIT_SCOPE(__FUNCTION__)
 
-namespace orbit::tracing {
+namespace orbit_tracing {
```

Comment:
Hmm here's a tricky case, this is in `OrbitBase` but was following a namespace logic of its own. Should this be `orbit_base` or `orbit_tracing` now? Does anybody have thoughts?

Reply:
If this is renamed to orbit_base we would probably want to rename Scope Listener and TimerCallback to TracingScope ...

Reply:
I guess it depends on a number of things: nesting policy, whether this merits a namespace or rather better to just remove and rename the structs, etc.

Reply:
I'd say, that following the discussion we had this week, our policy is `orbit_base` and than rename `Scope` to `TracingScope`,..., as Dimitry suggested.

Reply:
Not having been included in that discussion, I'm not sure I'm currently the best person to do that, but I can try.

Reply:
Basically, the idea would be to always use the module name (in snake case) as namespace and avoid nested namespaces. For free functions in the namespaces, ensure that the name is not ambiguous, and rather be more verbal on the name. I think it would be also fine to skip those cases if you are unsure.

Thanks for the change anyways.

Reply:
So I think we agreed that we should use `orbit_base` as namespace and then make the changes `Scope` -> `TracingScope`, `Listener` -> `TracingListener`. I guess `TimerCallback` can be moved inside `TracingListener`.

Reply:
I have a new commit ready with this change, will update the PR in a bit.